lib/rubocop/cop/generator.rb

```diff
@@ -117,7 +117,10 @@ def todo
       attr_reader :badge
 
       def write_unless_file_exists(path, contents)
-        raise "#{path} already exists!" if File.exist?(path)
+        if File.exist?(path)
+          $stderr.puts "rake new_cop: #{path} already exists!"
```

Comment:
I'd use `warn` here.

P.S. I thought we had some cop for this. :-)

Reply:
I see! First, I fixed it in the scope of this cop.